src/main/java/com/ericsson/ei/services/SubscriptionService.java

```diff
@@ -207,4 +220,27 @@ private String generateQuery(String subscriptionName, String ldapUserName) {
         }
         return query;
     }
+
+
+    /**
+     * This method finds whether a given subscription has an owner
+     *
+     * @param subscriptionName-
+     *            subscription name
+     * @return a boolean
+     * @throws SubscriptionNotFoundException
+     */
+
+    private boolean doSubscriptionOwnerExist(String subscriptionName) {
+        boolean ownerExist = true;
+        try {
+            if (getSubscription(subscriptionName).getLdapUserName()
+                                                 .isEmpty()) {
+                ownerExist = false;
+            }
+        } catch (SubscriptionNotFoundException e) {
+            LOG.error(e.getMessage());
```

Comment:
If an exception occurs, should ownerexist still be set to true? How can an owner exist when subscription could not be found?

Reply:
fixed